src/drivers/mct/shr/seq_timemgr_mod.F90

```diff
@@ -1019,26 +1029,23 @@ subroutine seq_timemgr_clockInit(SyncClock, nmlfile, restart, restart_file, pioi
           write(logunit,*) subname,' ERROR: offset not multiple',n,dtime(seq_timemgr_nclock_drv),offset(n)
           call shr_sys_abort()
        endif
-
        call ESMF_TimeIntervalSet( TimeStep, s=dtime(n), rc=rc )
        if(CurrTime + TimeStep > minStopTime ) then
           write(logunit,*) subname//" WARNING: Stop time too short, not all components will be advanced and restarts won't be written"
        endif
        if (n /= seq_timemgr_nclock_esp .and. trim(restart_option) .ne. &
             trim(seq_timemgr_optNone) .and. &
             trim(restart_option) .ne. trim(seq_timemgr_optNever)) then
-             write(logunit,*) subname, 'RestInterval=',minRestInterval,&
-                  ' Dtime=',dtime(n)
              call seq_timemgr_alarmGet(SyncClock%EAlarm(n,seq_timemgr_nalarm_restart), IntSec=RestInterval)
              if (RestInterval .ne. minRestInterval) then
                 write(logunit,*) subname, 'RestInterval=',RestInterval,&
                      ' minRestInterval=',minRestInterval
                 call shr_sys_abort("Component RestInterval inconsistant with driver")
              endif
-             if (mod(minRestInterval, dtime(n)) .ne. 0) then
-                write(logunit,*) subname, 'RestInterval=',minRestInterval,&
-                     ' Dtime=',dtime(n)
-                call shr_sys_abort("Restart interval not compatible with dtime")
+             if (RestInterval /= minRestInterval) then
```

Comment:
This check seems to now be duplicating the check a few lines up. Unless I'm missing something, one of these two blocks should be deleted.

Also, spelling fix: inconsistant should be inconsistent.